content/en/_index.html

```diff
@@ -0,0 +1,16 @@
++++
+title = "Goldydocs"
+linkTitle = "Goldydocs"
```

Comment:
i think maybe these should both be "tekton" as well? doesnt seem to make much difference but it does show up in the page source